ray-operator/test/e2eautoscaler/call_request_resources.py

```diff
@@ -0,0 +1,13 @@
+import ray
+import time
+import argparse
+from ray.autoscaler.sdk import request_resources
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--num-cpus", type=int, default=1)
+args = parser.parse_args()
+
+ray.init()
+request_resources(num_cpus=args.num_cpus)
```

Comment:
If I recall correctly, `request_resources` aren’t automatically released when the driver process exits. This is different from the normal tasks and actors.

That is, you don't need to check the number of nodes in this Python file. You can move the following loop to Golang.